src/coreclr/nativeaot/Runtime/arm/GcProbe.S

```diff
@@ -7,14 +7,16 @@
     .global RhpGcPoll2
 
     LEAF_ENTRY RhpGcPoll
-        PREPARE_EXTERNAL_VAR_INDIRECT_W RhpTrapThreads, 0
```

Comment:
Technically I can create similar macro for ARM

Reply:
I do not think that it is necessary.